packages/react/src/components/ContainedList/ContainedListItem/ContainedListItem.tsx

```diff
@@ -5,21 +5,30 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React from 'react';
+import React, { ElementType, ReactNode } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import { LayoutConstraint } from '../../Layout';
 import { usePrefix } from '../../../internal/usePrefix';
 
-function ContainedListItem({
+interface ContainedListItemProps {
+  action?: ReactNode;
+  children?: ReactNode;
+  className?: string;
+  disabled?: boolean;
+  onClick?: () => void;
+  renderIcon?: ElementType;
+}
```

Comment:
Same thing here with duplicating the comments from proptypes to up here.

Reply:
sure, will add comments